scipy/stats/tests/test_distributions.py

```diff
@@ -855,6 +870,21 @@ def test_extreme_values(self):
         assert_allclose(stats.logistic.ppf(1 - p), desired)
         assert_allclose(stats.logistic.isf(p), desired)
 
+    def test_logpdf_basic(self):
+        logp = stats.logistic.logpdf([-15, 0, 10])
+        # Expected values were computed with mpmath, using 50 digits
```

Comment:
Nit: I'd love if the comment fit on one line like

`# Expected values were computed with mpmath @ 50 digits`
or
`        # Expected values were computed with mpmath using 50 digits of precision` 
which is maybe exactly 80